File: CS/ViewModel/ViewModel.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WpfApplication41 {
    public class ViewModel {
        public virtual ObservableCollection<Customer> FixedTopRows {
            get;
            set;
        }
        public ObservableCollection<Customer> Customers {
            get;
            set;
        }

        public ViewModel() {
            Customers = new ObservableCollection<Customer>();
            for (int i = 1; i < 30; i++) {
                Customers.Add(new Customer() { ID = i, Name = "Name" + i });
            }

            FixedTopRows = new ObservableCollection<Customer>();
            FixedTopRows.Add(Customers[5]);
            FixedTopRows.Add(Customers[20]);
        }
    }

    public class Customer {
        public int ID {
            get;
            set;
        }

        public string Name {
            get;
            set;
        }
    }
}

File: VB/ViewModel/ViewModel.vb
Imports System.Collections.ObjectModel

Namespace WpfApplication41

    Public Class ViewModel

        Public Overridable Property FixedTopRows As ObservableCollection(Of Customer)

        Public Property Customers As ObservableCollection(Of Customer)

        Public Sub New()
            Customers = New ObservableCollection(Of Customer)()
            For i As Integer = 1 To 30 - 1
                Customers.Add(New Customer() With {.ID = i, .Name = "Name" & i})
            Next

            FixedTopRows = New ObservableCollection(Of Customer)()
            FixedTopRows.Add(Customers(5))
            FixedTopRows.Add(Customers(20))
        End Sub
    End Class

    Public Class Customer

        Public Property ID As Integer

        Public Property Name As String
    End Class
End Namespace
